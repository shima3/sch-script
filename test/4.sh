#!/bin/bash
$* sch-script.scm -e main sample/actor.sch
