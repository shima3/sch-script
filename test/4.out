s=(^ (f) f 0 (fibonacci 1 (+ 0 1)))
e=0
s=(^ (f) f 1 (fibonacci 1 (+ 1 1)))
e=1
s=(^ (f) f 1 (fibonacci 2 (+ 1 2)))
e=1
s=(^ (f) f 2 (fibonacci 3 (+ 2 3)))
e=2
s=(^ (f) f 3 (fibonacci 5 (+ 3 5)))
e=3
